packages/hui-pages/src/components/DocsHeader/DocsHeader.js

```diff
@@ -1,16 +1,13 @@
 import React from 'react'
-import { withRouter } from 'react-router-dom'
 import { Grid, Row, Col } from 'react-flexbox-grid'
 
 import Text from '@hixme-ui/text'
 
-const DocsHeader = ({
-  history
-}) => (
+const DocsHeader = () => (
   <Grid fluid style={{ padding: '0' }}>
     <Row style={{ margin: '20px 0 30px 0' }}>
       <Col>
-        <Text a light cursor='pointer' onClick={history.goBack}>
+        <Text a light href='/' cursor='pointer'>
```

Comment:
We may run into some problems when we deploy this to gh-pages, but let's wait and see.

Reply:
was wondering about this. It feels like the host root includes the hixme-ui route.